libc/src/math/generic/CMakeLists.txt

```diff
@@ -3941,6 +3941,55 @@ add_entrypoint_object(
     -O3
 )
 
+add_entrypoint_object(
+  totalordermag
+  SRCS
+    totalordermag.cpp
+  HDRS
+    ../totalordermag.h
+  DEPENDS
+    libc.src.__support.FPUtil.basic_operations
+  COMPILE_OPTIONS
+    -O3
+)
```

Comment:
Nit: the order should be `totalordermag`, `totalordermagf`, `totalordermagl`, `totalordermagf16`, `totalordermagf128`.